services/cd-service/pkg/repository/repository.go

```diff
@@ -676,6 +660,12 @@ type RootApp struct {
 type TransformerResult struct {
 	ChangedApps     []AppEnv
 	DeletedRootApps []RootApp
+	Commits         *Commits
+}
+
+type Commits struct {
+	Previous string
```

Comment:
I would find it more obvious if we would stick with *git.Oid here. Is there a specific reasons why you use string? If yes, maybe document it.

Reply:
done